README.md

```diff
@@ -16,6 +16,8 @@ See the website: https://scalameta.org/metals/
 
 ## Contributing
 
+We usually organize regular online events for contributors to meet up and work on Scala tooling, you can find more information on the [Scala Tooling Spree](https://scalameta.org/scala-tooling-spree/) page.
```

Comment:
Let's make it clear it's not limited to existing contributors :wink: 
```suggestion
We usually organize regular online events for contributors to meet up and work on Scala tooling, everyone is invited to join us! You can find more information on the [Scala Tooling Spree](https://scalameta.org/scala-tooling-spree/) page.
```

Reply:
Or even:
`We organize regular online events for anyone interested in contributing to Scala tooling.`
But I agree with Michał, it might sound a bit intimidating to newcomers the way it is now.

Reply:
Changed! I incorporated both or your comments 🎉